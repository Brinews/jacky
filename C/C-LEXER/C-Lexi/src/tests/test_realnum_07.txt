
  1,0

3.1415
