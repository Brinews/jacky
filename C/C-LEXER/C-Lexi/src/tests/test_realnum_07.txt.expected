{1 , NUM , 2}
{ , COMMA , 2}
{0 , NUM , 2}
{3.1415 , REALNUM , 4}
{ , END_OF_FILE , 6}
